website/content/docs/integrations/consul/index.mdx

```diff
@@ -174,3 +194,7 @@ Consul, with some exceptions.
 [int_consul_mesh]: /nomad/docs/integrations/consul/service-mesh
 [nomad_config_consul]: /nomad/docs/configuration/consul
 [service]: /nomad/docs/job-specification/service "Nomad service Job Specification"
+[bridge or CNI networking mode]: /nomad/docs/job-specification/network#mode
+[nameserver]: /nomad/docs/job-specification/network#servers
+[transparent proxy]: /nomad/docs/job-specification/transparent_proxy
+ [Forward DNS for Consul service discovery](https://developer.hashicorp.com/consul/tutorials/networking/dns-forwarding)
```

Comment:
I think this extra space is tripping up the Markdown rendering 😅 
https://nomad-git-docs-consul-dns-hashicorp.vercel.app/nomad/docs/integrations/consul#dns

Reply:
Crap, thanks. Will fix.

Reply:
https://github.com/hashicorp/nomad/pull/20387